app/src/test/java/org/simple/clinic/util/ErrorResolverTest.kt

```diff
@@ -131,6 +131,22 @@ class ErrorResolverTest {
     )
   }
 
+  @Test
+  @Parameters(value = ["403", "404", "500", "502"])
```

Comment:
Should `403` be considered as `Unexpected`?

Reply:
`403` is `HTTP Forbidden`. We currently do not explicitly handle `Forbidden` in the app, and it will be treated as a generic error message. This is why I left it as an `Unexpected` for now since we add a new type when needed.

Does this make sense?